src/main/resources/META-INF/reaktivity/mqtt.idl

```diff
@@ -74,12 +52,10 @@ scope mqtt
     {
         struct MqttBeginEx extends core::stream::Extension
         {
-            uint16 packetId;
             mqtt::MqttRole role;
             string clientId;
-            list<mqtt::MqttTopicFormat> topics;
-            list<mqtt::MqttReasonCode> reasonCodes;
-            list<mqtt::MqttUserProperty> properties;
+            string topic;
+            uint8 topicOptions;
```

Comment:
What exactly are these options? If modeled as flags, perhaps they could be captured in an `enum`?